### DISCUSSION TITLE
Local dependency?

### DISCUSSION THREAD

**Discussion by blaix** [gleam-lang/gleam#1530] [Questions & support] [answered]
I'm just starting with gleam and so far I'm loving it. Wonderful language.

Is there a way to point to a dependency on the local filesystem, or some other way to work on an in-development gleam package? How are people doing this now?

Thanks!

**Comment by lpil** [comment, marked as answer]
It's not something you can currently do, there's an issue you can follow here. https://github.com/gleam-lang/gleam/issues/1409

Currently I'm working on LSP so it won't happen soon unless someone picks it up. Some people are using submodules + symlinks to get this functionality for now.

**Reply by blaix** [reply]
Sounds good. Thanks for issue link. Looking forward to LSP!
